libs/labelbox/src/labelbox/client.py

```diff
@@ -2197,3 +2198,53 @@ def get_embedding_by_name(self, name: str) -> Embedding:
                 return e
         raise labelbox.exceptions.ResourceNotFoundError(Embedding,
                                                         dict(name=name))
+
+    def upsert_label_feedback(
+            self, label_id: str, feedback: str,
+            scores: Dict[str, float]) -> List[LabelScore]:
+        """
+        Submits the label feedback which is a free-form text and numeric
+        label scores.
+
+        Args:
+            label_id: Target label ID
+            feedback: Free text comment regarding the label
+            scores: A dict of scores, the key is a score name and the value is
+            the score value
+
+        Returns:
+            A list of LabelScore instances
+        """
+        mutation_str = """
+        mutation UpsertAutoQaLabelFeedbackPyApi(
```

Comment:
Since this our public doc (on readthedocs) please do not internal mutation code here. Rather, I would recommend to add some method usages

Reply:
> Since this our public doc (on readthedocs) please do not internal mutation code here

can you elaborate? 

> Rather, I would recommend to add some method usages

will add a usage example now 👍